src/tests/ftest/io/nvme_object.py

```diff
@@ -167,6 +167,12 @@ def setUp(self):
         self.pool_size = self.params.get("size", "/run/pool/createsize/*")
         self.array_size = self.params.get("array_size", "/run/container/*")
 
+        # update timeout for single_pool test case
+        timeout_for_single_pool = self.params.get(
+            "timeout_single_pool", "/run/timeouts/*/")
+        if "nvme_object_single_pool" in self.__str__():
+            self.timeout = timeout_for_single_pool
```

Comment:
Yeah, you are thinking right. We don't need the second param get()  bcoz it is being picked up by default due to the key 'timeout' in yams file.
Regarding your other suggestion, where we can make things more standardized, I think it will be interesting. Lets talk about it on Mon.